classes/classes/Saves.as

```diff
@@ -1792,18 +1792,28 @@ public function loadGameObject(saveData:Object, slot:String = "VOID"):void
 		player.hipRating = saveFile.data.hipRating;
 		player.buttRating = saveFile.data.buttRating;
 		
+		if (player.lowerBody > 3 && player.lowerBody < 5) { //Bitch's way of saying 4. So we can delete LOWER_BODY_TYPE_CENTAUR
```

Comment:
Uhm ... so you're literally saying, that you can't use the **'value'** 4 in the whole codebase anymore?

You removed the constant (LOWER_BODY_TYPE_CENTAUR), not the value.

PS: I'd suggest to leave the constant there forever so no one gets the idea to reuse lowerBody = 4 later. Same goes for LOWER_BODY_TYPE_DEERTAUR